docs/DeveloperGuide.md

```diff
@@ -283,28 +283,162 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `InterviewHub` and the **Actor** is the `hiring manager`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case: UC01 Add an interview**
 
 **MSS**
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1. Hiring manager requests to add an interview.
+2. InterviewHub adds the interview.
 
     Use case ends.
 
 **Extensions**
+* 1a. One of the user-provided parameters is invalid
+    * 1a1. InterviewHub shows an error message.
 
-* 2a. The list is empty.
+      Use case resumes at step 1.
 
-  Use case ends.
+**Use case: UC02 List all interview**
 
-* 3a. The given index is invalid.
+**MSS**
+
+1. Hiring manager requests to list all interviews.
+2. InterviewHub displays all scheduled interviews.
+
+    Use case ends.
+
+**Extensions**
+* 1a. The list is empty
+  * 1a1. InterviewHub shows an error message.
```

Comment:
Perhaps a more user-friendly way to handle listing out an empty list of upcoming interviews would be to print a nice error message such as "No interviews scheduled! Enjoy your day"(I'm assuming the engineering manager actually secretly hates interviewing people).

Reply:
Added to the week 8 meeting agenda as a discussion point on Use Case phrasing, good catch.